ionic2/app/components/poke-details/poke-attacks.component.scss

```diff
@@ -0,0 +1,7 @@
+span.vertical {
+  transform: rotate(270deg);
+  transform-origin: 85% 50%;
+  float: left;
+}
+
+
```

Comment:
fixed in https://github.com/PokemonGoers/Catch-em-all/pull/75/commits/7a627240f8710c2da745da2661c9b658b042c594